Clarify architecture overview (#4144)
Particular/docs.particular.net: platform/servicepulse.include.md

> ServicePulse is a web application aimed at administrators. It gives a clear, near real-time, high-level overview of how a system is currently functioning.

Reviewer: ```suggestion
ServicePulse is a web application designed for administrators. It provides a clear, near real-time, high-level overview of how a system is currently functioning.
```

> Notifications will occur when an endpoint is offline or when a message fails. [Custom checks](/monitoring/custom-checks/) can be built to get alerts for specific scenarios. The interface supports performing common operations for failure recovery, such as retrying failed messages. In order to display and handle them in a custom way, see [Subscribe to Publicly Exposed Events](/servicecontrol/contracts.md).

Reviewer: ```suggestion
Notifications will occur when an endpoint is offline or when a message fails. [Custom checks](/monitoring/custom-checks/) can be built to get alerts for specific scenarios. The interface supports performing common operations for failure recovery, such as retrying failed messages. In order to display and handle them in a custom way, see [Using ServiceControl Events](/servicecontrol/contracts.md).
```
